### PR TITLE
Customizable Feedback link

### PR DESCRIPTION
**Pull request by mpozhidaeva**
#544

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ define(['jquery', 'knockout', 'ohdsi.util', 'appConfig', 'webapi/AuthAPI', 'atla
 
 				return pageTitle;
 			});
+			self.supportURL = config.supportUrl;
```
Comment (anthonysena) [inline js/app.js]: Where is 'config.supportUrl' defined? Is this part of another PR?

Comment (mpozhidaeva) [reply js/app.js]: Sorry for not clear dependencies between PRs.
**supportUrl** is a property in config.json. Please see https://github.com/OHDSI/Atlas/pull/553/files#diff-6a02156604c06becbe5fdca178766acc (line 10)

**Comment by anthonysena** [comment]
@mpozhidaeva - can you please combine the contents of this PR with #561 for review? Thanks!

**Comment by anthonysena** [comment]
Closing as this functionality is now covered in #564. We'll review and approve from that PR.

**Pull request closed without merging**